src/mpi/coll/ineighbor_alltoall/ineighbor_alltoall.c

```diff
@@ -204,7 +207,47 @@ int MPIR_Ineighbor_alltoall_impl(const void *sendbuf, int sendcount,
     MPIR_Sched_t s = MPIR_SCHED_NULL;
 
     *request = NULL;
+    /* If the user picks one of the transport-enabled algorithms, branch there
+     * before going down to the MPIR_Sched-based algorithms. */
+    /* TODO - Eventually the intention is to replace all of the
+     * MPIR_Sched-based algorithms with transport-enabled algorithms, but that
+     * will require sufficient performance testing and replacement algorithms. */
+    if (comm_ptr->comm_kind == MPIR_COMM_KIND__INTRACOMM) {
+        /* intracommunicator */
+        switch (MPIR_Ineighbor_allgatherv_intra_algo_choice) {
+            case MPIR_INEIGHBOR_ALLTOALL_INTRA_ALGO_GENTRAN_LINEAR:
```

Comment:
Should the choice variable be `..._alltoall_...`?

Reply:
Yes. I fixed it.